django_freeradius/tests/test_admin.py

```diff
@@ -213,3 +213,15 @@ def test_nas_admin_save_model(self):
         self.assertNotContains(response, 'error')
         nas.refresh_from_db()
         self.assertEqual(nas.type, 'my-custom-type')
+
+    def test_cleanup_stale_radacct_command(self):
```

Comment:
I only realized now the tests for the management commands shouldn't go in `test_admin.py`. Please move all the tests you created recently to a new file named `test_commands.py`.
You should do this change in a separate commit than the current one but you can leave both in the current PR

Reply:
https://github.com/openwisp/django-freeradius/issues/35